truffleruby/src/main/ruby/core/rbconfig.rb

```diff
@@ -78,50 +77,124 @@ module RbConfig
       'LIBRUBYARG' => '',
       'prefix' => '',
       'ruby_install_name' => ruby_install_name,
+      'RUBY_INSTALL_NAME' => ruby_install_name,
+      'ruby_version' => ruby_version,
+      'RUBY_BASE_NAME' => ruby_base_name,
       'RUBY_SO_NAME' => '$(RUBY_BASE_NAME)'
   }
 
+
+  CONFIG = {
```

Comment:
Since there is only the `RUBY_SO_NAME` as substitution above, how about
```
CONFIG = MAKEFILE_CONFIG.dup
CONFIG['RUBY_SO_NAME'] = ruby_base_name
```
?